Examples/KerrBH/params_cheap.txt

```diff
@@ -0,0 +1,106 @@
+#Params for runtime inputs
+
+verbosity = 0
+chk_prefix = KerrBH_
+plot_prefix = KerrBHp_
+#restart_file = KerrBH_000060.3d.hdf5
+
+# NB - the N values need to be multiples of block_factor
```

Comment:
It is a very good idea to be able to specify the full length params too.
 
I would propose N_full -> N_full_length? This is a bit more descriptive (I tried to think of a better short name, but can't). I think we should also add a comment here to describe what it does to help users.

Reply:
Do you think adding `_length` is necessary to these parameter names is necessary if it's explained what it is in a comment? In some sense, many of the parameter names are somewhat meaningless without further information/explanation e.g. `L`.

Reply:
Sure, I am happy with just a comment. The name full_length means more to me but I agree it is still not evident... N_without_symmetries? I'd be happier with a clearer name, but it is fine as long as it is commented.

Reply:
N was 1 character. That's insanely long Katy :P haha